db/migration/migrations/000023_add_metadata_indexes.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+CREATE INDEX IF NOT EXISTS metadata_requested_version_idx ON fhir_endpoints_metadata(requested_fhir_version);
```

Comment:
We actually already have a migration 23, though it's currently only on the main branch. So these should be 24, and I can merge in the main branch changes to develop